packages/storybook-angular/src/stories/CheckboxStates.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import { Meta, Story } from '@storybook/angular';
+import { UtrechtCheckboxAttr } from '@utrecht/component-library-angular';
+import checkboxMeta from './Checkbox.stories';
+
+export default {
+  title: 'Angular Component/Checkbox/States',
+  id: 'angular-checkbox--states',
+  component: UtrechtCheckboxAttr,
+  argTypes: checkboxMeta.argTypes,
+} as Meta<typeof UtrechtCheckboxAttr>;
+
+const Template: Story = (args) => ({
+  props: args,
+  template: `<input type="checkbox" [disabled]="disabled" [checked]="checked" [value]="value" utrecht-checkbox />`,
+  component: UtrechtCheckboxAttr,
+});
+
+export const Checked = Template.bind({});
+
+Checked.args = {
+  checked: true,
+};
+
+export const CheckedAndDisabled = Template.bind({});
+
+CheckedAndDisabled.args = {
+  checked: true,
+  disabled: true,
+};
+
+export const Disabled = Template.bind({});
+
+Disabled.args = {
+  disabled: true,
+};
+
+export const Hover = Template.bind({});
+
+Hover.parameters = {
+  pseudo: { hover: true },
+};
+
+export const Focus = Template.bind({});
+
+Focus.parameters = {
+  pseudo: { focus: true, focusVisible: true },
+};
```

Comment:
I suggest we put all the checkbox stories in one file, `Checkbox.stories.tsx`.

I think I mainly want to use separate files for stories that combine multiple components, instead of showing of the atomic component.